packages/plugins/src/api/index.js

```diff
@@ -77,6 +143,21 @@ export function registerPlugin( name, settings ) {
  *
  * @param {string} name Plugin name.
  *
+ * @example <caption>ES5</caption>
+ * ```js
+ * // Using ES5 syntax
```

Comment:
It seems like `caption` and code comment duplicate each other. Is there an easy way to generate the comment or paragraph to simplify it?

Reply:
Yeah. As I mentioned, the idea will be that the caption is used for this, but it's not implemented at the moment.